torch/onnx/symbolic.py

```diff
@@ -579,29 +576,25 @@ def symbolic(g, input, all_weights, h0, **fkwargs):
         h_outs = h_out if num_layers == 1 else g.op('Concat', *h_outs, axis_i=0)
         return prev_output, h_outs
 
-    return torch.onnx.symbolic_override(symbolic)
+    return symbolic
 
 
 def LSTM_symbolic_builder(input_size, hidden_size, num_layers, batch_first, dropout, bidirectional, **kwargs):
-    def symbolic(g, input, all_weights, h0_and_c0, **fkwargs):
-        if batch_first:
-            return _unimplemented("LSTM", "batch_first")
-        if dropout:
-            return _unimplemented("LSTM", "dropout")
-        if bidirectional:
-            return _unimplemented("LSTM", "bidirectional")
+    if batch_first:
```

Comment:
move it back to `symbolic`. In my understanding we want to throw inside `symbolic` to get slightly better error messages attributed to a particular node

Reply:
++, I think this code got changed underneath me

Reply:
Yep, I changed it :)